apps/odyssey-stats/webpack.config.js

```diff
@@ -22,6 +22,7 @@ const shouldEmitStats = process.env.EMIT_STATS && process.env.EMIT_STATS !== 'fa
 const isDevelopment = process.env.NODE_ENV !== 'production';
 const outBasePath = process.env.STATS_PACKAGE_PATH ? process.env.STATS_PACKAGE_PATH : __dirname;
 const outputPath = path.join( outBasePath, 'dist' );
+const sourceMap = isDevelopment ? 'source-map' : false;
```

Comment:
For now, enabling this for development mode only, but I believe this could be enabled by default